modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -190,26 +190,37 @@ Using random access, you can change a record's payload at any given index positi
+and improves an operation's load. Batch jobs use Mule runtime's thread pools, so
+there is no default for the job. Each thread iterates through that block to
+process each record, and then each block is queued back, and the process continues.
 
-Consider having 1 million records to place in a queue for a 3 step batch job. At least three million I/O operations occur as the runtime takes and requests each record as they move through the job's phases. +
-Performance requires having enough available memory to process the 16 threads in parallel, which means moving 1600 records from persistent storage into RAM. The larger your records and their quantity, the more available memory you need for batch processing.
+Consider having 1 million records to place in a queue for a 3-step batch job. At least three million I/O operations occur as the runtime takes and requests each record as they move through the job's phases. +
+Performance requires having enough available memory to process the threads in parallel, which means moving the records from persistent storage into RAM. The larger your records and their quantity, the more available memory you need for batch processing.
 
-Although, the standard model of 16 threads, with 100 records per batch job works for most use cases, consider three use cases where you might need to increase or decrease the block size:
+Although the standard model of up to 100 records per thread in the batch job works for most use cases, consider three use cases where you need to increase or decrease the block size:
 
 * Assume you have 200 records to process through a batch job. With the default 100-record block size, Mule can only process two records in parallel at a time. If you request fewer than 101 records, then your processing becomes sequential. If you need to process heavy payloads, then queueing a hundred records demands a large amount of working memory.
-* Consider a batch job that needs to process images, and an average image size of 3 MB. You then have 100 blocks with payloads of 3 MB, being processed in 16 threads. Hence your default threading-profile setting would require around 4.6 GB of working memory just to keep the blocks in the queue. You should set a lower block size to distribute each payload through more jobs and lessen the load on your available memory.
-* Suppose having 5 million records with payloads so small that you can fit blocks of 500 records in your memory without problems. Setting a larger block size improves your batch job time without sacrificing working memory load.
+* Consider a batch job that needs to process images, and an average image size of 3 MB. You then have 100-record blocks with payloads of 3 MB, being processed in your threads. Hence, your default threading-profile setting would require a large amount of working memory just to keep the blocks in the queue. In this case, set a lower block size to distribute each payload through more jobs and lessen the load on your available memory.
```

Comment:
You then have 100-record blocks with payloads of 3 MB, being processed in your threads. >
In this scenario, Mule processes 100-record blocks with payloads of 3 MB in each thread.

Is this better? To avoid the passive voice.